.jfrog/frogbot-config.yaml

```diff
@@ -0,0 +1 @@
+- repoName: frogbot
```

Comment:
I think it is super important to make sure that Frogbot works also without this file. Otherwise, it will make the starter workflows impossible to use.
```suggestion
```

Reply:
Required only for Bitbucket Server. Works without the config file in GitLab and GitHub.